pandas/tests/series/test_timeseries.py

```diff
@@ -370,6 +370,16 @@ def test_pct_change(self, datetime_series):
             rs, (filled / filled.shift(freq="5D") - 1).reindex_like(filled)
         )
 
+    def test_pct_change_with_duplicate_axis(self):
+        # GH 28664
+        original_idx = date_range("2019-11-14", periods=3, freq="D")
+        s = Series(range(3), original_idx)
+
+        result = s.pct_change(freq="B")
+        expected = s.asfreq("B").pct_change().reindex(original_idx)
```

Comment:
Can you construct `expected` from literal values? Right now we are using `pct_change` in both the `result` and `expected`, which makes it unclear that this actually works as intended

Reply:
You're right, this is not explicit obviously. Thanks, I edited it.